tools/scripts/build_projects.py

```diff
@@ -239,7 +239,7 @@ def build(self):
 			err = run_cmd(cmd + ' reset')
 			if err != 0:
 				return err
-			err = run_cmd_stm("timeout 200s " + cmd + ' VERBOSE=y -j%d all' % (multiprocessing.cpu_count() - 1))
+			err = run_cmd_stm("timeout 200s " + cmd + ' VERBOSE=y -j%d all' % ((multiprocessing.cpu_count() - 1)/2))
```

Comment:
let's skip the `- 1` then, just divide by 2 if you think "all but one" cores is too much